tutorials/bash-christmas-tree/index.mdx

```diff
@@ -123,4 +123,11 @@ ssh root@<Instance Public IP>
     The script writes an animated tree on the terminal window:
 
     <Lightbox src="scaleway-animated-tree.gif" alt="" />
-2. Exit the script by pressing `CTRL` + `c`.
\ No newline at end of file
+2. Exit the script by pressing `CTRL` + `c`.
+
+<ClickableBanner
+  productLogo="generic"
+  title="Bring some holiday fun to your terminal with this Bash script!"
```

Comment:
It's hard to sell an advantage of creating a SCW account for this tutorial really, as anyone with a computer can already achieve the tutorial and there's no point to creating a SCW resource to do so. Potentially consider removing CTA from this one? I guess it depends.

Reply:
I agree with Rowena - also this tutorial is quite useless 
We might consider to remove it.